www-cronrobot/app/javascript/stylesheets/main-dashboard.scss

```diff
@@ -1,28 +1,233 @@
+$bg-color: #222121;
+$main-blue: #5A6EF8;
```

Comment:
When you edit a scheduler, it should be Update, instead of Create

![image](https://user-images.githubusercontent.com/8007722/132255540-3a6b9b49-83ae-4a33-bc80-ea0612f69865.png)